inspirehep/dojson/experiments/rules.py

```diff
@@ -48,56 +48,56 @@ def date_started(self, key, value):
             self['date_started'] = val.get('s')
         if val.get('t'):
             self['date_completed'] = val.get('t')
+        if val.get('c'):
+            self['date_cancelled'] = value.get('c')
```

Comment:
We were actually  missing this xd (it's not a change introduced by the bump, but a previous missing field)